plugins/sigma/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.19...3.24 FATAL_ERROR)
+
+project(
+  sigma
+  DESCRIPTION "Sigma query language plugin for Tenzir"
+  LANGUAGES CXX)
```

Comment:
Why is this not a builtin?

Reply:
I plan to eventually make this depend on libfswatch, so I think a plugin makes more sense than a builtin.